client/src/components/relatedItems/SingleCard.jsx

```diff
@@ -131,27 +135,51 @@ export default class SingleCard extends React.Component {
   }
   render() {
     return (
-      // when client click card, it will redirect page to detail product page
-      <div class="card" onClick={()=> { window.location.href = `http://localhost:3000/?id=${this.props.id}`}}>
-
-        <img class="card-img" src={this.state.img} alt="Product image"
-        // onError={ event => {
-        //   event.target.src= "https://cdn.pixabay.com/photo/2015/01/21/13/21/sale-606687__340.png"
-        //   event.onerror = null
-        // }}
-        />
-
-        <div class="product-info">
-          {/* icon is not designed */}
-          {<p>{this.props.icon}</p>}
-          < ComparisonMedal currentProductFeature={this.props.currentProductFeature} relatedProductFeature={this.state.features} />
-
-          <span class="category">{this.state.category}</span>
-          <br></br>
-          <span class="name">{this.state.name}</span>
 
-          {(this.state.discount_price) ? <p class="discount-price">${this.state.discount_price}</p> : null}
-          <p class="default-price" style={(this.state.discount_price) ? {"text-decoration": "line-through"} : null}>${this.state.default_price}</p>
+
+
+       // when client click card, it will redirect page to detail product page
+      <div className="card" >
+
+        <div className="img-container">
+          <img className="card-img" src={this.state.img} alt="Product image" onClick={()=> { window.location.href = `http://localhost:3000/?id=${this.props.id}`}}
+          // onError={ event => {
+          //   event.target.src= "https://cdn.pixabay.com/photo/2015/01/21/13/21/sale-606687__340.png"
+          //   event.onerror = null
+          // }}
+          />
+
+          {/* icon is star now */}
+          {this.props.icon === 'star' ?
+            <div className="icon-star" >
+              <button onClick={this.onCompare}><span className="star">&#9733;</span></button>
+
+              {/* comparisonMedal appear or not */}
+              { this.state.isCompareOn ?
+              <Modal className="modal">
+                < ComparisonModal currentProductFeature={this.props.currentProductFeature} relatedProductFeature={this.state.features} />
+              </Modal>
+                : null}
+
+            </div>
+
+            : null
+          }
+
+
+        </div>
+
+
+
+
+        <div style={{"marginLeft": "5px"}}>
+
```

Comment:
I always think it's best to keep styles inside the CSS sheet. Maybe put a className on the div and keep the style in the styles.css.

Reply:
OK. I think it's the best practice of css. I'll change it in next pull request